packages/govuk-frontend/src/govuk/components/footer/_index.scss

```diff
@@ -73,15 +70,12 @@
 
   .govuk-footer__copyright-logo {
     display: inline-block;
-    min-width: $govuk-footer-crest-image-width;
-    padding-top: ($govuk-footer-crest-image-height + govuk-spacing(2));
+    min-width: $govuk-footer-crest-width;
+    padding-top: ($govuk-footer-crest-height + govuk-spacing(2));
     background-image: govuk-image-url("govuk-crest.png");
```

Comment:
Could you do a quick ImageOptim (or similar) run on the PNG image?

I'm seeing another 8.9% file size saving (which is great)

Reply:
Oh wow. Hadn’t thought to do that also (I used ImageMagick to reduce colour palette only). Have updated the image, it’s now 7,624 bytes. 😄